main.js

```diff
@@ -17,4 +17,168 @@ mobileMenuItems.forEach((i) => {
+    tags: ['html', 'Ruby on rails', 'css', 'javascript'],
+    popupTags: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+    image: 'pics/homepage4.png',
+    desktopImage: 'pics/homepage4-desktop.png',
+    seeLive: '#',
+    seeSource: '#',
+  },
+];
+
+function subtitleValue(i) {
+  let subtitleList = [];
+  for (let j = 0; j < data[i].subtitle.length; j += 1) { subtitleList += `<li class="subtitle-${j}"> ${data[i].subtitle[j]} </li>`; }
+  return (subtitleList);
+}
+
+function tagsValue(i) {
+  let tagsList = [];
+  for (let j = 0; j < data[i].tags.length; j += 1) { tagsList += `<li class="tags-${j}"> ${data[i].tags[j]} </li>`; }
+  return tagsList;
+}
+
+function popupTagsValue(i) {
+  let tagsList = [];
+  for (let j = 0; j < data[i].tags.length; j += 1) { tagsList += `<li class="tool"> ${data[i].tags[j]} </li>`; }
+  return tagsList;
+}
+
+for (let i = 0; i < data.length; i += 1) {
+  document.querySelector('.work-container').innerHTML += `
+<div id="work-${i}" class="work-tonic">
+<div class="work-img-div">
+        <img src="${data[i].image}" alt="Project image" class="work-img"/>
+        <img src="${data[i].desktopImage}" alt="Project image" class="desk-work-img"/>
+      </div>
+      <div class="work-info">
+        <h2 class="tonic">${data[i].title}</h2>
+        <div class="text-info">
+          <ul>
+           ${subtitleValue(i)}
+          </ul>
+        </div>
+        <p class="work-text"> ${data[i].desription} </p>
+          <ul class="tags">
+            ${tagsValue(i)}
+              </ul>
+        
+        <button type="button" id="${data[i].id}" class='see-proj see-proj-${i}'>See project</button>
+      </div>
+    </div>`;
+}
+
+const arr = [];
+
+for (let i = 0; i < data.length; i += 1) {
+  arr.push(` <div class="modal">
+  <div class="popup-header">
+  <h2 class="tonic">${data[i].title}</h2>
+  <div id="close-popup-sign">
+      <span> <a  href="#" class="close-popUp">X</a></span>
+    </div>
+    </div>
+    <div class="subtitlePopupValue text-info">
+      <ul>
+        ${subtitleValue(i)}
+      <ul>
+    </div>
+  <div class="modal-div-3">
+    <div class="modal-img">
+    <img class="work-img1-mob" src="${data[i].image}" alt="work-img1" />
+    <img class="work-img1-desk" src="${data[i].desktopImage}" alt="work-img1" />
+    </div>
+    <div class="modal-info">
+    
+      <p class="modal-description desk mob">
+     ${data[i].popDescription}
+      </p>
+
+    <div class="tech">
+    <ul class="modal-tools desk">
+    ${popupTagsValue(i)}
+      </ul>
+      <ul class="modal-tools mobile">
+      ${popupTagsValue(i)}
+      </ul>
+      <button type="button" class="modal-btn">
+      See Live <i class="fa fa-external-link" aria-hidden="true"></i>
+      </button>
+      <button type="button" class="modal-btn">
+      See Source
+      <i class="fa fa-github-square" aria-hidden="true"></i>
+      </button>
```

Comment:
- [ ]  The buttons that should sit side-by-side on the same line so your design matches the Figma design. kindly work on that.
- Your design:
<img width="737" alt="image" src="https://user-images.githubusercontent.com/77190203/156466785-07852ada-52e4-4b97-bafd-8b05070c9d14.png">
<img width="203" alt="image" src="https://user-images.githubusercontent.com/77190203/156466855-a44637cc-3417-4b6c-aa12-36ff856416ac.png">

- Figma design:
<img width="470" alt="image" src="https://user-images.githubusercontent.com/77190203/156467003-27a05a96-7d8e-4c2e-a371-4ffdfb849270.png">

<img width="152" alt="image" src="https://user-images.githubusercontent.com/77190203/156466945-882a5a8d-647e-44a5-999c-3f1712939b46.png">

Reply:
Thanks for your feedback. I have worked on that. Cheers